src/pages/workspace/card/WorkspaceCardPage.tsx

```diff
@@ -1,36 +1,29 @@
-import PropTypes from 'prop-types';
+import type {RouteProp} from '@react-navigation/native';
 import React from 'react';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import WorkspacePageWithSections from '@pages/workspace/WorkspacePageWithSections';
 import CONST from '@src/CONST';
 import WorkspaceCardNoVBAView from './WorkspaceCardNoVBAView';
 import WorkspaceCardVBANoECardView from './WorkspaceCardVBANoECardView';
 import WorkspaceCardVBAWithECardView from './WorkspaceCardVBAWithECardView';
 
-const propTypes = {
-    /** The route object passed to this page from the navigator */
-    route: PropTypes.shape({
-        /** Each parameter passed via the URL */
-        params: PropTypes.shape({
-            /** The policyID that is being configured */
-            policyID: PropTypes.string.isRequired,
-        }).isRequired,
-    }).isRequired,
-
-    ...withLocalizePropTypes,
+/** Defined route object that contains the policyID param, WorkspacePageWithSections is a common component for Workspaces and expect the route prop that includes the policyID */
+type WorkspaceCardPageProps = {
+    route: RouteProp<{params: {policyID: string}}>;
 };
```

Comment:
```suggestion
type WorkspaceCardPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.CARD>;
```

Reply:
I initially started with this type but WorkspacePageWithSection doesn't allow this type. This page WorkspacePageWithSections used the withPolicy HOC which makes this type not compatible.
Do you have any idea how to overcome this kind of issues?